man/systemd-debug-generator.xml

```diff
@@ -70,6 +70,39 @@
     <citerefentry><refentrytitle>systemd.generator</refentrytitle><manvolnum>7</manvolnum></citerefentry>.</para>
   </refsect1>
 
+  <refsect1>
+    <title>System Credentials</title>
+
+    <variablelist class='system-credentials'>
+      <varlistentry>
+        <term><varname>systemd.unit.*</varname></term>
```

Comment:
maybe call this `systemd.extra-unit.*`?

i.e. these are "extra", i.e. not part of the OS itself, but added on top. we kind use the word "extra" quite often for things that are "side-loaded" like this, i.e. are secondary extension stuff to the primary way to do things. i#d stick to that naming here.